platform/ui/src/components/studyList/TableSearchFilter.js

```diff
@@ -77,10 +77,18 @@ function TableSearchFilter(props) {
 
   return translationsAreReady
     ? meta.map((field, i) => {
-        const { displayText, fieldName, inputType } = field;
+        const {
+          displayText,
+          fieldName,
+          inputType: fieldInputType,
+          clearable,
+        } = field;
         const isSortField = sortFieldName === fieldName;
         const sortIcon = isSortField ? sortIconForSortField : sortIcons[0];
-
+        let inputType = fieldInputType;
+        if (clearable && inputType === 'text') {
+          inputType = 'search';
+        }
```

Comment:
I'm not sure I follow what's going on here.

Reply:
At user experience: add x icon to remove text on a input field
At dev level: use inputType as search when field is clearable